### DISCUSSION TITLE
Relative URLs to absolute URLs?

### DISCUSSION THREAD

**Discussion by sramirezpch** [nuxt-modules/i18n#1314] [Q&A] [answered]
Hi

I did a SEO test (sitechecker.pro) and, among other results, I got one realted to relative URLs. Is there a way to generate an absolute URL with `:to="localePath('/relative/path/here')"`?

**Comment by rchl** [comment, marked as answer]
`baseUrl` option - https://i18n.nuxtjs.org/options-reference#baseurl
